packages/postgres-database/src/simcore_postgres_database/models/funcapi_function_jobs_table.py

```diff
@@ -0,0 +1,77 @@
+"""Function jobs table
+
+- List of function jobs served by the simcore platform
+"""
+
+import uuid
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+
+from ._common import RefActions, column_created_datetime, column_modified_datetime
+from .base import metadata
+from .funcapi_functions_table import functions_table
+
+function_jobs_table = sa.Table(
+    "funcapi_function_jobs",
+    metadata,
+    sa.Column(
+        "uuid",
+        UUID(as_uuid=True),
+        primary_key=True,
+        index=True,
+        default=uuid.uuid4,
+        doc="Unique id of the function job",
+    ),
+    sa.Column(
+        "title",
+        sa.String,
+        doc="Name of the function job",
+    ),
+    sa.Column(
+        "description",
+        sa.String,
+        doc="Description of the function job",
+    ),
+    sa.Column(
+        "function_uuid",
+        sa.ForeignKey(
+            functions_table.c.uuid,
+            onupdate=RefActions.CASCADE,
+            ondelete=RefActions.CASCADE,
+            name="fk_function_jobs_to_function_uuid",
+        ),
+        nullable=False,
+        index=True,
+        doc="Unique identifier of the function",
+    ),
+    sa.Column(
+        "function_class",
+        sa.String,
+        doc="Class of the function",
+    ),
+    sa.Column(
+        "status",
+        sa.String,
+        doc="Status of the function job",
+    ),
+    sa.Column(
+        "inputs",
+        sa.JSON,
+        doc="Inputs of the function job",
+    ),
+    sa.Column(
+        "outputs",
+        sa.JSON,
+        doc="Outputs of the function job",
+    ),
+    sa.Column(
+        "class_specific_data",
+        sa.JSON,
+        nullable=True,
+        doc="Fields specific for a function class",
+    ),
```

Comment:
postgres has json and jsonB... in the past we had to convert to jsonB. maybe worth checking before you put this in.